Example/PinpointKitExample/AppDelegate.swift

```diff
@@ -11,10 +11,10 @@ import PinpointKit
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
-
-    var window: UIWindow? = ShakeDetectingWindow(frame: UIScreen.main.bounds)
     
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
+    var window: UIWindow?
```

Comment:
Mind explaining the window change. I vaguely remember hearing about it, but forget.